src/main/java/parser/FileParser.java

```diff
@@ -12,20 +12,19 @@ public FileParser(String str){
 
     public Task lineToTask(){
 
-        String[] words = this.thisLine.split(" | ");
+        String[] words = this.thisLine.split(" \\| ");
```

Comment:
Can consider using regular expressions instead! That will greatly ease the burden of parsing strings as commands (see Java 11 Patterns)